float32 x_estimated_
float32 y_estimated_
uint64 time_stamp_
float32 vx_estimated_
float32 vy_estimated_
float32 x_gt_
float32 y_gt_
float32 vx_gt_
float32 vy_gt_
